src/config/dynamic-theme-fixes.config

```diff
@@ -12150,6 +12150,22 @@ pre {
 
 ================================
 
+patreon.com
+
+INVERT
+a[href^="https://docs.google.com"] img
+a[href^="https://forms.gle"] img
+
+CSS
+div[data-tag="media-container"] > img {
+    filter: brightness(75%);
+}
+
+IGNORE IMAGE ANALYSIS
+*
```

Comment:
Why should this be disabled?

Reply:
Mostly because the avatars are inverted, which causes unsightly, bright white spots on the page.

Reply:
I should mention I tried a lot of different css selector ways to target just the avatar images, but patreon randomizes the css classes and doesn't use any id tags.

Reply:
If nothing otherwise breaks then LGTM for this case.